src/context/auth.tsx

```diff
@@ -2,21 +2,29 @@ import React, {
   createContext,
   useContext,
   FunctionComponent,
-  useState
+  useState,
+  useEffect
 } from "react";
+import { AsyncStorage } from "react-native";
+
+export const SESSION_TOKEN_KEY = "SESSION_TOKEN";
+export const EXPIRY_KEY = "EXPIRY_KEY";
+export const ENDPOINT_KEY = "ENDPOINT_KEY";
 
 interface AuthenticationContext {
-  authKey: string;
+  token: string;
+  expiry: string;
```

Comment:
Would storing this as a number be better? So consumers don't need to convert yet again

Reply:
yeah true